test/jasmine-proxy.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I don't quite get why do we need this separated file. Maybe we could use one `jasmine.json` and add command line param to test core/duplex/proxy